### ISSUE TITLE
[Feature request] Enforce spaces on string concatenations

### ISSUE THREAD

**Issue by acelaya** [slevomat/coding-standard#198]
It would be great to have a sniff that allows to force a specific number of spaces between elements when concatenating strings.

For example. Enforcing 0 spaces;

```php
$world = 'World';
echo 'Hello '.$world.'!'; // Correct
echo 'Hello ' . $world . '!'; // Incorrect
```

Enforcing 1 space;

```php
$world = 'World';
echo 'Hello '.$world.'!'; // Incorrect
echo 'Hello ' . $world . '!'; // Correct
```

Is there any possibility to have this sniff? :-)

Great work, by the way.

**Comment by kukulich** [comment]
The sniff is part as PHPCS itself, see https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/Squiz/Sniffs/Strings/ConcatenationSpacingSniff.php

**Comment by acelaya** [comment]
Nice!!
Thanks :-)
